src/server/mod.rs

```diff
@@ -61,15 +64,14 @@ impl Server {
         self.state.reload(watcher)
     }
 
-    fn handle_client_message(&mut self, com: Communication) {
-        use crate::shared::message::Message;
+    fn handle_client_message(&mut self, message: Message) {
```

Comment:
Maybe ensure that we send `Com::End` at the end without relaying that `state` will do it ?